app/src/docs/v1.api-spec.yaml

```diff
@@ -270,18 +443,16 @@ components:
         body:
           type: string
           description: >-
-            The body of the message as an Unicode string, Buffer, Stream or an
-            attachment-like object
-          example: Hello World content
-        cc:
+            A Nunjucks compatible body template of the message as an Unicode
```

Comment:
Let's not say stream or buffer, keep it simple - it's just a string (same as in email).  Even though our underlying infrastructure can handle stream etc, we would need to test our API can handle that. 

Also, let's not say it's nunjucks here, maybe we should add in the other field that the caller declares the template language - with nunjucks as our only (currently) supported enum.

Reply:
I added a new optional field called `engine` with an enum and default value of `nunjucks`. Also dropped references to stream and buffer to only strings.